packages/lib/src/vis/line/LineVis.tsx

```diff
@@ -28,12 +28,18 @@ import styles from './LineVis.module.css';
 import type { TooltipData } from './models';
 import { CurveType } from './models';
 
-// Default line colors in the following format: `[<light-mode>, <dark-mode>]`
 // Inspired by Matplotlib palette: https://matplotlib.org/stable/gallery/color/named_colors.html
-const DEFAULT_CURVE_COLOR = ['darkblue', 'deepskyblue'];
-const DEFAULT_AUX_COLORS = [
-  'orangered, forestgreen, red, mediumorchid, olive',
-  'orange, lightgreen, red, violet, gold',
+const COLORS: CustomColor[] = [
+  {
+    property: '--h5w-line--color',
+    fallback: 'darkblue',
+    darkFallback: 'deepskyblue',
+  },
+  {
+    property: '--h5w-line--colorAux',
+    fallback: 'orangered, forestgreen, red, mediumorchid, olive',
+    darkFallback: 'orange, lightgreen, red, violet, gold',
+  },
```

Comment:
I've made the API of `useCustomColors` a lot less obscure. Let me know what you think.